ichnaea/data/tasks.py

```diff
@@ -82,6 +82,11 @@ def monitor_queue_size(self):
     monitor.QueueSize(self)()
 
 
+@celery_app.task(base=BaseTask, bind=True, queue="celery_monitor")
+def sentry_test(self, msg):
+    monitor.SentryTest(self, msg=msg)()
```

Comment:
This could be a regular function call (``self.raven_client.captureMessage(msg)``) rather than a callable class instance. It would break the pattern of the other tasks, but I think that would be a good thing.

Reply:
I like this idea. +1